### PR TITLE
Clarified some of the getter-setter instructions. 

### LINKED ISSUE #37612: Use getters and setters to Control Access to an Object unclear instructions
First I want to thank everyone who works on freeCodeCamp. I have learned a lot from it so far. 

That said, some of the instructions for some of the exercises are quite poor and in desperate need of improvement. A perfect example of this issue is the ES6 exercise "Use getters and setters to Control Access to an Object."

This exercise tells the new user that "Getter functions are meant to simply return (get) the value of an object's private variable to the user without the user directly accessing the private variable." 

However, the solution to this exercise requires that the user create a getter that alters the private variable in the constructor. This may be a perfectly valid use of a getter in JavaScript but it feels very misleading when the user has just been told that a getter is for the purpose of retrieving a variable that is meant to be private (I assume so that the variable is not accidentally altered, hoisted, etc.). If there are, indeed, other valid uses of a getter the exercise should address them or at least provide a link to some external documentation (or something) that explains this.

The solution I am referencing is found here: https://www.freecodecamp.org/forum/t/freecodecamp-challenge-guide-use-getters-and-setters-to-control-access-to-an-object/301220

The instructions to the user say that he/she should create a class, a getter, and a setter to convert Fahrenheit to Celsius. The instructions do not tell the user that there should be a way to convert from Celsius to Fahrenheit, despite the fact that the solution requires exactly that.

There are clues, but to call them such would be generous at best and I did not recognize them as such until I already knew the solution. For instance, the instructions give the formula for conversion of Celsius to Fahrenheit, albeit with no clear directive attached. Also, there is the "Note" section where the following text appears:

"When you implement this, you would be tracking the temperature inside the class in one scale - either Fahrenheit or Celsius.

This is the power of getter or setter - you are creating an API for another user, who would get the correct result, no matter which one you track."

This note is difficult to parse, grammatically awkward, and terribly unclear. What's all this about "tracking" things, for instance? If this is supposed to inform the user that the class should be able to convert temperatures both ways, it should be entirely rewritten.

I consider myself pretty good at reading! I am a college educated, native English speaker. I hold a law degree - I know how to speak, read, and write the English language. If I find exercises like this confusing and misleading I can only imagine that others, especially non-native speakers, must also be baffled by instructions like these.

Thank you again for your hard work and the valuable service that freeCodeCamp provides. I hope this issue will be taken in the constructive way in which it is intended.

### PR DESCRIPTION
**Pull request by Pallof**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to #37612

I changed around some of the variable names to remove the 'jokiness' aspect.
There were some comments in regards to how 'tracking' had a sense of poor understanding. I did not touch this, because I wasn't sure how to best reword what we were aiming at.
There were also mentions of changing the solutions around, but nothing decisive, so I left it as is.

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md
Comment (Sky020) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md:44]: A lot of confusion comes from users being given the formula for temperature conversion after the instruction to create a `getter`. With what you have added makes it sound like:
> Create a constructor that converts an input Fahrenheit into Celcius.
> Create a getter that converts Fahrenheit into Celcius, and
> create a setter that converts Celcius into Fahrenheit.

I would suggest a lot more of the challenge be re-worded, ensuring a decent workflow be implemented.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md
Comment (Sky020) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md:12]: >Getter functions are meant to access...

I would change this to:
> Getter functions return...

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
In general, I am struggling to suggest changes, because most users completely brush over _getters_ and _setters_, without doing further research anyway. The way this lesson and challenge comes across is: Getters and setters exist, and act almost identically to an object method. Whilst, really, in Javascript, getters and setters have such specific use cases and implementation.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md
Comment (Pallof) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md:44]: @Sky020 okay, Ill make an edit to it tonight and try to change the wording around

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md
Comment (tinktink1) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md:12]: I think that these changes will really help! When I first started learning how to program earlier this year everything was a little bit confusing for me. I think your updates will help clear everything up for people.

**Comment by nhcarrigan** [comment]
@Pallof Can you provide an update on your implementation of the requested changes?

**Comment by bhagerty** [comment]
I have proposed a different solution over on [the issue page that led to this PR](https://github.com/freeCodeCamp/freeCodeCamp/issues/37612#issuecomment-682992990). I have not yet made a PR, but I'm willing to try if the community agrees.

**Comment by moT01** [comment]
Thanks for the PR @Pallof 🎉 I believe that edits from maintainers are being blocked on this. So, in the interest of saving time, I have made a new PR for this. Thanks and happy coding 🎉

**Pull request closed without merging**
